apps/web/src/services/api/profiles/change-profile-username.ts

```diff
@@ -0,0 +1,18 @@
+import { supabase } from '@/services/supabase'
```

Comment:
It is better to use the back-end term (`update`) instead of change to maintain consistency and greater clarity